src/figuresClasses.ts

```diff
@@ -1,19 +1,103 @@
+enum Colors {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
 export interface Figure {
+  shape: Shape;
+  color: Colors;
+  getArea(): number;
+}
+
+function isNotValidTriangle(...shapeSide: number[]): boolean {
+  const sumSides = shapeSide
+    .reduce((sideOne: number, sideTwo: number) => sideOne + sideTwo);
 
+  return shapeSide.some((side: number) => side >= (sumSides - side));
 }
 
-export class Triangle {
+function isValidLength(...lengths : number[]): boolean {
+  return Math.min(...lengths) <= 0;
+}
```

Comment:
what? Why -1 is valid?

<img width="171" alt="image" src="https://user-images.githubusercontent.com/86183007/224666020-c3b6aa0b-3d08-4f45-8d24-a8a7b974a76e.png">